libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/NavigationRouteEx.kt

```diff
@@ -194,17 +192,17 @@ private fun List<LegStep>.updateSteps(
     return result
 }
 
-private fun DirectionsResponse.Builder.updateWaypoints(
+private fun buildNewWaypoints(
```

Comment:
I should have asked this before, but I noticed this functionality only now.
What kind of update in waypoints do we expect? We don't expect to receive waypoints with different coordinates, do we?

Reply:
Done.